Tiltfile

```diff
@@ -488,6 +484,7 @@ def deploy_clusterclass(clusterclass_name, label, filename, substitutions):
     cmd_button(
         clusterclass_name + ":apply",
         argv = ["bash", "-c", apply_clusterclass_cmd],
+        env = dictonary_to_list_of_string(substitutions),
```

Comment:
Q: Does this mean the `env` field of cmd_button has a different format than the one for `local_resource`?
(Looks like this transformation was not necessary above in l.478

Reply:
Yes, I tried to pass the `substitutions` directly and it failed with an error `Error in v1alpha1.cmd: v1alpha1.cmd: for parameter "env": value should be a List or Tuple of strings, but is of type dict`

The official documents suggest the same ([cmd_button env](https://api.tilt.dev/core/cmd-v1alpha1.html#:~:text=Expressed%20as%20a%20C%2Dstyle%20array%20of%20strings%20of%20the%20form%20%5B%E2%80%9CKEY1%3DVALUE1%E2%80%9D%2C%20%E2%80%9CKEY2%3DVALUE2%E2%80%9D%2C%20%E2%80%A6%5D.)) that `env` parameter of `cmd_button` requires a list of strings of format `"name=value"`. Eg: ["env1=val1", "env2=val2"]

Reply:
Sounds good. Thank you!